pkg/workload/main.go

```diff
@@ -0,0 +1,203 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
Seems this is a binary for the test. Why not put it into the `tests/workload` directory?

Reply:
Since QE team want to use this workload to do the TiCDC perfromance testing, and I think it's natural to put it in `/pkg` if another repo will import it.  Do you think it is better to put it into `tests/workload`?